lib/facil/tls/fio_tls_missing.c

```diff
@@ -19,7 +19,7 @@ Feel free to copy, use and enjoy according to the license provided.
  */
 #include "fio_tls.h"
 
-#if 1 /* TODO: place library compiler flags here */
+#if !defined(HAVE_TLS) /* Library compiler flags */
```

Comment:
The conservative approach was exactly right.

However - namespaces:

Please rename the flag to `FIO_TLS_FOUND`. This will mitigate possible name conflicts.

Reply:
Thanks! There is no prefix because this flag is only used during build, not exported to users, I followed the surrounding style (`HAVE_OPENSSL`, `HAVE_BEARSSL`, etc.), but will push a new commit that renames this flag shortly.

Reply:
Perfect, thanks for the quick fix :)

FYI: the `HAVE_*` style is the `configure` utility style. I kept it only for library/feature names that could be share across libraries. It's unlikely, but it might conflict with other flags when multiple libraries are used (in the `./lib` folder).